addons/point_of_sale/tests/test_anglo_saxon.py

```diff
@@ -51,7 +51,8 @@ def setUp(self):
         self.cash_journal = self.env['account.journal'].create({'name': 'CASH journal', 'type': 'cash', 'code': 'CSH00'})
         self.sale_journal = self.env['account.journal'].create({'name': 'SALE journal', 'type': 'sale', 'code': 'INV00'})
         self.pos_config.invoice_journal_id = self.sale_journal
-        self.pos_config.journal_ids = [self.cash_journal.id]
+        # 'pos.config' object has no attribute 'journal_ids'
+        # self.pos_config.journal_ids = [self.cash_journal.id]
```

Comment:
```suggestion
```
was correctly handled in https://github.com/odoo/odoo/commit/8becb17b8a3aa18b92629c4ddd33630ea4b1444e